go.mod

```diff
@@ -139,3 +141,5 @@ require (
 )
 
 replace github.com/golang/glog => github.com/nginxinc/glog v1.1.2
+
+replace github.com/nginxinc/telemetry-exporter => /Users/s.odonovan/github.com/nginxinc/telemetry-exporter
```

Comment:
Will remove this line as soon as https://github.com/nginxinc/telemetry-exporter/pull/27 is merged

Reply:
https://github.com/nginxinc/telemetry-exporter/pull/27 is merged